docs/tutorial/part-two/index.md

```diff
@@ -72,26 +72,26 @@ You should now have a structure like this:
 │       └── global.css
 ```
 
-2. Define some styles in the `global.css` file:
+2. `Global.css` फ़ाइल में कुछ सीएसएस स्टाइल्स को परिभाषित करें:
```

Comment:
`परिभाषित` => `डिफाइन`

I am not very sure about this. You can take a call on this, if this makes sense, otherwise we can ignore this